### PR TITLE
feat(frontend): create map for TokenUi

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

For better readability and reusability, we create a function to map an object `Token` to an object `TokenUi`.

### CODE REVIEW

File: src/frontend/src/lib/utils/token.utils.ts [hunk 1]
```diff
@@ -88,3 +88,28 @@ export const calculateTokenUsdBalance = ({
...
 };
+
+/** Maps a Token object to a TokenUi object, meaning it adds the balance and the USD balance to the token.
+ *
+ * @param token - The given token.
+ * @param $balancesStore - The balances data for the tokens.
+ * @param $exchanges - The exchange rates data for the tokens.
+ * @returns The token UI.
+ */
+export const mapTokenUi = ({
```
Comment (peterpeterparker) [inline src/frontend/src/lib/utils/token.utils.ts:99]: Should we add a test since it's a utils?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/utils/token.utils.ts (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/utils/token.utils.ts:99]: i thought of that, but it is actually just a map, in the sense that it actually has no logic inside, just an aggregation of other functions (that have tests). Maybe it is not an util? is there a map folder?

Comment (peterpeterparker) [reply src/frontend/src/lib/utils/token.utils.ts:99]: If has some logic, it calls `calculateTokenUsdBalance` and map a balance only if the related balance is defined.

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/utils/token.utils.ts:99]: right! included

**Pull request merged**